### ISSUE TITLE
Jupyter notebook not finding correct packages

### ISSUE THREAD

**Issue by Zylatis** [pypa/pipenv#4154]
I have a, as far as i can tell, functioning pipenv install using the following lockfile

```[[source]]
name = "pypi"
url = "https://pypi.org/simple"
verify_ssl = true

[dev-packages]

[packages]
pandas = "==1.0.1"
boto3 = "==1.9.126"
lifetimes = "*"
pytz = "*"
mysql-connector-python="==8.0.6"
XlsxWriter="==1.2.6"
fr8xpy = {path = "./fr8xpy-0.1.0-py2.py3-none-any.whl"}
loguru="*"
xlrd="*"
jupyter="*"
ipykernel = "*"

[requires]
python_version = "3.6"
```

For example when I go into a pipenv shell and run a python shell in the terminal, i can run the above packages but i cannot run those not listed there (which are also not dependencies) so this all seems correct.

However, when I run jupyter notebooks I run into a problem. In particular it refuses to import my local package fr8xpy. If, from within that notebook, i use !pip list then I see what i run in terminal from within the pipenv shell, which is indeed different to my 'global' package list.

I have verified this by trying to import a package in the global list which is missing from the pipenv list. I am able to import this from a global shell, but not from python running in pipenv, nor from python running in jupyter in pipenv, so this all makes sense.

I therefore don't understand why I cannot import fr8xpy? It simply says module not found. This works fine from a python shell in pipenv (even if i temporarily remove it from the global package list so i know it's not just reading that somehow)

I should also note that I made a specific kernel based off the pip env environment based on a SO comment using
```
python -m ipykernel install --user --name=pricing_project
```
from within pipenv

**Comment by amhrasmussen** [comment]
Hi @Zylatis 
I've successfully used pipenv+jupyter and local .whl previously, so I suspect this is not actually a bug in pipenv. To help others help you with this, try to repeat you process with a .whl of one of the packages that are working for you, e.g. pandas.

Also, do read this https://github.com/pypa/pipenv/blob/master/CONTRIBUTING.md, particularly this section https://github.com/pypa/pipenv/blob/master/CONTRIBUTING.md#bug-reports.

**Comment by Zylatis** [comment]
Hi @amhrasmussen great idea, will do and will report back. Apologies if post in wrong spot/format.

**Comment by matteius** [comment]
I believe this should be resolved on `pipenv==2022.8.19` -- can this be closed @Zylatis ?
